app/docker/components/imageRegistry/por-image-registry.html

```diff
@@ -46,7 +46,7 @@
     <div class="form-group">
       <span class="small">
         <p class="text-muted" style="margin-left: 15px">
-          <i class="fa fa-exclamation-circle blue-icon" aria-hidden="true" style="margin-right: 2px"></i>
+          <pr-icon feather="true" icon="'alert-circle'" style="margin-right: 2px"></pr-icon>
```

Comment:
change all related style to classes

Reply:
please remove all in-line style( margin-right: 2px ), use bootstrap or tailwindcss instead.